spec-tests/src/test/java/org/candlepin/spec/pools/PoolSpecTest.java

```diff
@@ -565,4 +571,61 @@ public void shouldIncludeProductBrandingWhenFetchingPool() {
         assertTrue(b2filter.isPresent());
     }
 
+    @Test
+    @OnlyInStandalone
+    @DisplayName("should includes null values in json in Standalone")
+    public void shouldIncludesNullValuesInJsonInStandalone() throws ApiException {
+        ApiClient adminClient = ApiClients.admin();
+        OwnerDTO owner = this.createOwner(adminClient);
+        ApiClient ownerClient = this.createUserClient(adminClient, owner);
```

Comment:
The owner client is extraneous here -- we're not interested in how the fetch is scoped.